src/main/java/karuna/karuna_backend/Repositories/ContentRepository.java

```diff
@@ -0,0 +1,14 @@
+package karuna.karuna_backend.Repositories;
+
+import karuna.karuna_backend.Models.Content;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface ContentRepository extends JpaRepository<Content, Long> {
+    public Optional<List<Content>> findByPage(String page);
```

Comment:
Im not sure, but probalby it alaways be not null, just an empty list

Reply:
Good point, changed.